app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecord.kt

```diff
@@ -0,0 +1,46 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord
+
+import androidx.room.Dao
+import androidx.room.Embedded
+import androidx.room.Entity
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy.REPLACE
+import androidx.room.PrimaryKey
+import androidx.room.Query
+import androidx.room.Relation
+import org.simple.clinic.drugs.PrescribedDrug
+import java.util.UUID
+
+@Entity
+data class TeleconsultRecord(
+    @PrimaryKey
+    val id: UUID,
+
+    val patientId: UUID,
+
+    val medicalOfficerId: UUID,
+
+    @Embedded(prefix = "request_")
+    val teleconsultRequestInfo: TeleconsultRequestInfo?,
+
+    @Embedded(prefix = "record_")
+    val teleconsultRecordInfo: TeleconsultRecordInfo?,
```

Comment:
Shouldn't the type here be `TeleconsultRecordWithPrescribedDrugs`?

Reply:
No, `TeleconsultRecordInfo` contains record information while the `TeleconsultRecordWithPrescribedDrugs` is a table that will have teleconsultRecordId and prescribedDrugUuid